Milestone 8: Mobile Menu

GuilhermeGiachellin opened the pull request:
-Created a menu with JavaScript
-Used Dom to do it

GuilhermeGiachellin committed:
Create scrip.js

-Added the menu when click the hamburguer

GuilhermeGiachellin committed:
Update html and style

-Added needed changes
-Style to the menu

GuilhermeGiachellin committed:
Added assets

GuilhermeGiachellin committed:
Linters correction

GuilhermeGiachellin committed:
Lynter Correction

GuilhermeGiachellin committed:
Lynters

GuilhermeGiachellin committed:
Linters errors

GuilhermeGiachellin committed:
Linter error correction

GuilhermeGiachellin committed:
Linter errors

GuilhermeGiachellin committed:
Memory overflow

-The i + 1 inside of the for is not working, so I used i++ again.

GuilhermeGiachellin committed:
Linter errors

GuilhermeGiachellin committed:
Linter error

GuilhermeGiachellin committed:
Linter errors

GuilhermeGiachellin committed:
Linter errors

GuilhermeGiachellin committed:
Linter error

GuilhermeGiachellin committed:
Linter correction

vichuge commented on scrip.js:
```diff
@@ -0,0 +1,64 @@
+});
+
+document.querySelectorAll('a[href^="#"]').forEach((anchor) => {
+  anchor.addEventListener('click', function (e) {
```
- [x] I kindly invite you to use ES6 in every part of your script, this following the [Javascript best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md) 🐶

GuilhermeGiachellin replied:
Thank you for the review! 

About the ES6, I believe we not about ES6 so I don't know how to adapt. Everything else is corrected.

vichuge replied:
I agree with you, don't worry :)

vichuge reviewed (changes requested):
# Required Changes ♻️ ⚠️ 🚦 

Hi dear student, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![mario_ladders](https://user-images.githubusercontent.com/20563252/121369265-4e418880-c901-11eb-848c-1eff06d5a4cf.gif)

## Good points 👍 
- No Linters error ✔️

## Aspects to improve ♻️

- [x] It looks like the open menu doesn't cover the entire screen, I kindly invite you to fix it.
![image](https://user-images.githubusercontent.com/20563252/122849371-ff1c3000-d2d0-11eb-8150-f33291dab2c3.png)

- [x] At desktop size the navbar appears as moved to the right side, making it impossible to click the options on the navbar.
![image](https://user-images.githubusercontent.com/20563252/122849503-42769e80-d2d1-11eb-91b7-7b28a701f85d.png)

- [x] Check the comments under the review.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

GuilhermeGiachellin committed:
Review correction

-Corrected all the mistakes that were present.
-ES6 is going to be viewed in next module.

GuilhermeGiachellin committed:
Linter error

vichuge reviewed (approved):
# Approved project! ✅ 🥳 🎈 🎉🎉🎉🎉🎉🎉🎉🎉 

Hi there, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

![InbornCandidHornet-size_restricted](https://user-images.githubusercontent.com/20563252/121368858-ff93ee80-c900-11eb-9ee8-e43b2b0d3382.gif)

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

GuilhermeGiachellin committed:
Update to the scrool function
